### ISSUE TITLE
No calibration samples are taken

### ISSUE THREAD

**Issue by NXTler** [linux-surface/iptsd#177]
Hello,
I have a Surface 5 Pro and have had many issues with the touch being registered while writing and some gestures not working at all. During a calibration attempt, I noticed that small touches (like for a zoom gesture) are registered (seen by the timer running), but no samples are taken, only when I press harder.

**Comment by otjamin** [comment]
Does https://github.com/linux-surface/iptsd/issues/171#issuecomment-2359625900 solve your problem?

**Comment by NXTler** [comment]
No, the touch is detected (seen by the timer running), but no samples are taken while calibrating. Changing these values doesn't fix the issue.

**Comment by otjamin** [comment]
> No, the touch is detected (seen by the timer running), but no samples are taken while calibrating. Changing these values doesn't fix the issue.

Okay, sorry I couldn't help, but I had the same behavior when calibrating with the bad values. Timer was going on every touch, but only very strong touches influenced the calibration values.

**Comment by dingodoppelt** [comment]
> > No, the touch is detected (seen by the timer running), but no samples are taken while calibrating. Changing these values doesn't fix the issue.
> 
> Okay, sorry I couldn't help, but I had the same behavior when calibrating with the bad values. Timer was going on every touch, but only very strong touches influenced the calibration values.

The issue persists but the calibration tool doesn't seem to be the issue. It counts Samples on `contact.stable` which I tried setting to `contact.valid` without any success. Funny enough that without iptsd the one-finger-touch behaviour is near perfect but missing all the good stuff. Quite unusable at this moment in time
